library/src/main/AndroidManifest.xml

```diff
@@ -15,6 +15,15 @@
         <activity android:name=".view.activity.AttachmentDocumentActivity" />
         <activity android:name=".view.activity.AttachmentActivity" />
 
+        <receiver
```

Comment:
@elevenetc is it normal to have this in the library code?

Reply:
It is. 
It doesn't affect apps and just default android way to implement broadcast events distribution among app components. Also it's safe and can't be called by other apps (exported="false")